src/accumulator.rs

```diff
@@ -122,8 +126,8 @@ pub enum State<Candidate, Digest, Signature> {
 #[derive(Debug, Default)]
 #[cfg_attr(any(test, feature="codec"), derive(Encode, Decode))]
 struct VoteCounts {
-	prepared: usize,
-	committed: usize,
+	prepared: u32,
```

Comment:
Why not use `u64`?

Reply:
usually if we know it won't overflow u32 then we usually prefer u64, so that simple arithmetic can convert from it easily. I suspect to be the case here, but I don't really know